evm/ts/package.json

```diff
@@ -44,12 +44,17 @@
     "test": "jest --config ./jest.config.ts"
   },
   "dependencies": {
-    "@wormhole-foundation/sdk-connect": "0.6.5",
+    "@wormhole-foundation/sdk-connect": "0.7.1-beta.2",
     "@wormhole-foundation/sdk-definitions-ntt": "0.1.0-beta.0",
-    "@wormhole-foundation/sdk-evm": "0.6.5",
-    "@wormhole-foundation/sdk-evm-core": "0.6.5",
+    "@wormhole-foundation/sdk-evm": "0.7.1-beta.2",
+    "@wormhole-foundation/sdk-evm-core": "0.7.1-beta.2",
     "ethers": "^6.5.1"
   },
+  "peerDependencies": {
+    "@wormhole-foundation/sdk-connect": "0.7.1-beta.2",
+    "@wormhole-foundation/sdk-evm": "0.7.1-beta.2",
+    "@wormhole-foundation/sdk-evm-core": "0.7.1-beta.2"
```

Comment:
These and above should be reverted to the more flexible `^0.7` before merging and a `0.7.1` release should be published to latest tag